### ISSUE TITLE
The whole wayfire session crashes while closing a window if the `animate` plugin is disabled

### ISSUE THREAD

**Issue by zhangkaizhao** [WayfireWM/wayfire#1017]
**Describe the bug**
The whole wayfire session crashes while closing a window if the `animate` plugin is disabled.

**To Reproduce**
1. Disable plugin `animate`
2. Start Wayfire from tty
3. Start an application e.g. alacritty
4. Close the current window (e.g. of alacritty)

**Expected behavior**
The whole wayfire session does not crash.

**Screenshots or stacktrace**
File content of `wf.log` of running `wayfire > wf.log` from tty:
```
II 09-01-21 14:44:49.996 - [wayfire-0.6.0/src/main.cpp:325] Starting wayfire version 0.6.0-177fb87 (Jan  8 2021, branch 'master')
II 09-01-21 14:44:50.015 - [backend/session/logind.c:853] Successfully loaded logind session
II 09-01-21 14:44:50.019 - [backend/backend.c:157] Found 1 GPUs
II 09-01-21 14:44:50.019 - [backend/drm/backend.c:138] Initializing DRM backend for /dev/dri/card0 (radeon)
II 09-01-21 14:44:50.019 - [backend/drm/drm.c:255] Found 6 DRM CRTCs
II 09-01-21 14:44:50.019 - [backend/drm/drm.c:176] Found 6 DRM planes
II 09-01-21 14:44:50.068 - [render/egl.c:297] Using EGL 1.5
II 09-01-21 14:44:50.068 - [render/egl.c:298] Supported EGL client extensions: EGL_EXT_device_base EGL_EXT_device_enumeration EGL_EXT_device_query EGL_EXT_platform_base EGL_KHR_client_get_all_proc_addresses EGL_EXT_client_extensions EGL_KHR_debug EGL_EXT_platform_device EGL_EXT_platform_wayland EGL_KHR_platform_wayland EGL_EXT_platform_x11 EGL_KHR_platform_x11 EGL_MESA_platform_gbm EGL_KHR_platform_gbm EGL_MESA_platform_surfaceless
II 09-01-21 14:44:50.068 - [render/egl.c:299] Supported EGL display extensions: EGL_ANDROID_blob_cache EGL_EXT_buffer_age EGL_EXT_create_context_robustness EGL_EXT_image_dma_buf_import EGL_EXT_image_dma_buf_import_modifiers EGL_KHR_cl_event2 EGL_KHR_config_attribs EGL_KHR_create_context EGL_KHR_create_context_no_error EGL_KHR_fence_sync EGL_KHR_get_all_proc_addresses EGL_KHR_gl_colorspace EGL_KHR_gl_renderbuffer_image EGL_KHR_gl_texture_2D_image EGL_KHR_gl_texture_3D_image EGL_KHR_gl_texture_cubemap_image EGL_KHR_image EGL_KHR_image_base EGL_KHR_image_pixmap EGL_KHR_no_config_context EGL_KHR_reusable_sync EGL_KHR_surfaceless_context EGL_EXT_pixel_format_float EGL_KHR_wait_sync EGL_MESA_configless_context EGL_MESA_drm_image EGL_MESA_image_dma_buf_export EGL_MESA_query_driver EGL_WL_bind_wayland_display 
II 09-01-21 14:44:50.068 - [render/egl.c:300] EGL vendor: Mesa Project
II 09-01-21 14:44:50.070 - [render/gles2/renderer.c:674] Using OpenGL ES 3.1 Mesa 20.3.2
II 09-01-21 14:44:50.070 - [render/gles2/renderer.c:675] GL vendor: X.Org
II 09-01-21 14:44:50.070 - [render/gles2/renderer.c:676] GL renderer: AMD TURKS (DRM 2.50.0 / 5.10.5-arch1-1, LLVM 11.0.0)
II 09-01-21 14:44:50.070 - [render/gles2/renderer.c:677] Supported GLES2 extensions: GL_EXT_blend_minmax GL_EXT_multi_draw_arrays GL_EXT_texture_filter_anisotropic GL_EXT_texture_compression_s3tc GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_format_BGRA8888 GL_OES_compressed_ETC1_RGB8_texture GL_OES_depth24 GL_OES_element_index_uint GL_OES_fbo_render_mipmap GL_OES_mapbuffer GL_OES_rgb8_rgba8 GL_OES_standard_derivatives GL_OES_stencil8 GL_OES_texture_3D GL_OES_texture_float GL_OES_texture_float_linear GL_OES_texture_half_float GL_OES_texture_half_float_linear GL_OES_texture_npot GL_OES_vertex_half_float GL_EXT_draw_instanced GL_EXT_texture_sRGB_decode GL_OES_EGL_image GL_OES_depth_texture GL_AMD_performance_monitor GL_OES_packed_depth_stencil GL_EXT_texture_type_2_10_10_10_REV GL_NV_conditional_render GL_OES_get_program_binary GL_APPLE_texture_max_level GL_EXT_discard_framebuffer GL_EXT_read_format_bgra GL_EXT_frag_depth GL_NV_fbo_color_attachments GL_OES_EGL_image_external GL_OES_EGL_sync GL_OES_vertex_array_object GL_OES_viewport_array GL_ANGLE_pack_reverse_row_order GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_EXT_occlusion_query_boolean GL_EXT_robustness GL_EXT_texture_rg GL_EXT_unpack_subimage GL_NV_draw_buffers GL_NV_read_buffer GL_NV_read_depth GL_NV_read_depth_stencil GL_NV_read_stencil GL_EXT_draw_buffers GL_EXT_map_buffer_range GL_KHR_debug GL_KHR_robustness GL_KHR_texture_compression_astc_ldr GL_NV_pixel_buffer_object GL_OES_depth_texture_cube_map GL_OES_required_internalformat GL_OES_surfaceless_context GL_EXT_color_buffer_float GL_EXT_sRGB_write_control GL_EXT_separate_shader_objects GL_EXT_shader_implicit_conversions GL_EXT_shader_integer_mix GL_EXT_tessellation_point_size GL_EXT_tessellation_shader GL_EXT_base_instance GL_EXT_compressed_ETC1_RGB8_sub_texture GL_EXT_copy_image GL_EXT_draw_buffers_indexed GL_EXT_draw_elements_base_vertex GL_EXT_gpu_shader5 GL_EXT_polygon_offset_clamp GL_EXT_primitive_bounding_box GL_EXT_render_snorm GL_EXT_shader_io_blocks GL_EXT_texture_border_clamp GL_EXT_texture_buffer GL_EXT_texture_cube_map_array GL_EXT_texture_norm16 GL_EXT_texture_view GL_KHR_context_flush_control GL_KHR_robust_buffer_access_behavior GL_NV_image_formats GL_OES_copy_image GL_OES_draw_buffers_indexed GL_OES_draw_elements_base_vertex GL_OES_gpu_shader5 GL_OES_primitive_bounding_box GL_OES_sample_shading GL_OES_sample_variables GL_OES_shader_io_blocks GL_OES_shader_multisample_interpolation GL_OES_tessellation_point_size GL_OES_tessellation_shader GL_OES_texture_border_clamp GL_OES_texture_buffer GL_OES_texture_cube_map_array GL_OES_texture_stencil8 GL_OES_texture_storage_multisample_2d_array GL_OES_texture_view GL_EXT_blend_func_extended GL_EXT_buffer_storage GL_EXT_float_blend GL_EXT_geometry_point_size GL_EXT_geometry_shader GL_KHR_no_error GL_KHR_texture_compression_astc_sliced_3d GL_OES_EGL_image_external_essl3 GL_OES_geometry_point_size GL_OES_geometry_shader GL_OES_shader_image_atomic GL_EXT_clip_cull_distance GL_EXT_disjoint_timer_query GL_EXT_texture_compression_s3tc_srgb GL_MESA_shader_integer_functions GL_EXT_clip_control GL_EXT_color_buffer_half_float GL_EXT_texture_compression_bptc GL_KHR_parallel_shader_compile GL_EXT_EGL_image_storage GL_EXT_texture_sRGB_R8 GL_MESA_framebuffer_flip_y GL_EXT_depth_clamp GL_EXT_texture_query_lod 
II 09-01-21 14:44:50.075 - [wayfire-0.6.0/src/main.cpp:362] using config file: /home/kaizhao/.config/wayfire.ini
II 09-01-21 14:44:50.075 - [wf-config-0.6.0/src/file.cpp:445] Reading XML configuration options from directory /usr/share/wayfire/metadata
II 09-01-21 14:44:50.075 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/workarounds.xml
II 09-01-21 14:44:50.076 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/wobbly.xml
II 09-01-21 14:44:50.076 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/place.xml
II 09-01-21 14:44:50.076 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/animate.xml
II 09-01-21 14:44:50.076 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/window-rules.xml
II 09-01-21 14:44:50.076 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/core.xml
II 09-01-21 14:44:50.076 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/command.xml
EE 09-01-21 14:44:50.076 - [wf-config-0.6.0/src/xml.cpp:147] Could not parse /usr/share/wayfire/metadata/command.xml: option at line 10 has no default value specified.
II 09-01-21 14:44:50.076 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/alpha.xml
II 09-01-21 14:44:50.076 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/idle.xml
II 09-01-21 14:44:50.076 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/move.xml
II 09-01-21 14:44:50.076 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/cube.xml
II 09-01-21 14:44:50.077 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/grid.xml
II 09-01-21 14:44:50.077 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/autorotate-iio.xml
II 09-01-21 14:44:50.077 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/force-fullscreen.xml
II 09-01-21 14:44:50.077 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/blur.xml
II 09-01-21 14:44:50.077 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/vswitch.xml
II 09-01-21 14:44:50.077 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/wrot.xml
II 09-01-21 14:44:50.078 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/autostart.xml
EE 09-01-21 14:44:50.078 - [wf-config-0.6.0/src/xml.cpp:147] Could not parse /usr/share/wayfire/metadata/autostart.xml: option at line 10 has no default value specified.
II 09-01-21 14:44:50.078 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/fast-switcher.xml
II 09-01-21 14:44:50.078 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/follow-focus.xml
II 09-01-21 14:44:50.078 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/crosshair.xml
II 09-01-21 14:44:50.078 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/mag.xml
II 09-01-21 14:44:50.078 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/annotate.xml
II 09-01-21 14:44:50.078 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/water.xml
II 09-01-21 14:44:50.078 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/background-view.xml
II 09-01-21 14:44:50.078 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/resize.xml
II 09-01-21 14:44:50.078 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/vswipe.xml
II 09-01-21 14:44:50.078 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/showrepaint.xml
II 09-01-21 14:44:50.078 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/extra-gestures.xml
II 09-01-21 14:44:50.079 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/keycolor.xml
II 09-01-21 14:44:50.079 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/simple-tile.xml
II 09-01-21 14:44:50.079 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/fisheye.xml
II 09-01-21 14:44:50.079 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/zoom.xml
II 09-01-21 14:44:50.079 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/input.xml
II 09-01-21 14:44:50.079 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/invert.xml
II 09-01-21 14:44:50.079 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/decoration.xml
II 09-01-21 14:44:50.079 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/bench.xml
II 09-01-21 14:44:50.079 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/oswitch.xml
II 09-01-21 14:44:50.079 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/scale.xml
II 09-01-21 14:44:50.080 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/join-views.xml
II 09-01-21 14:44:50.080 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/wm-actions.xml
II 09-01-21 14:44:50.080 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/expo.xml
II 09-01-21 14:44:50.080 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/switcher.xml
II 09-01-21 14:44:50.080 - [wf-config-0.6.0/src/file.cpp:456] Reading XML configuration options from file /usr/share/wayfire/metadata/workspace-names.xml
II 09-01-21 14:44:50.081 - [backend/noop/backend.c:51] Creating noop backend
II 09-01-21 14:44:50.081 - [backend/noop/backend.c:16] Starting noop backend
II 09-01-21 14:44:50.083 - [xwayland/server.c:92] WAYLAND_SOCKET=18 Xwayland :0 -rootless -terminate -listen 16 -listen 17 -wm 20
II 09-01-21 14:44:50.090 - [wayfire-0.6.0/src/main.cpp:382] Using socket name wayland-1
II 09-01-21 14:44:50.125 - [wayfire-0.6.0/src/core/seat/input-manager.cpp:43] handle new input: Power Button, default mapping: (null)
II 09-01-21 14:44:50.131 - [wayfire-0.6.0/src/core/seat/input-manager.cpp:43] handle new input: Video Bus, default mapping: (null)
II 09-01-21 14:44:50.138 - [wayfire-0.6.0/src/core/seat/input-manager.cpp:43] handle new input: Lid Switch, default mapping: (null)
II 09-01-21 14:44:50.138 - [wayfire-0.6.0/src/core/seat/input-manager.cpp:43] handle new input: Sleep Button, default mapping: (null)
II 09-01-21 14:44:50.144 - [wayfire-0.6.0/src/core/seat/input-manager.cpp:43] handle new input: ASUS USB2.0 WebCam: ASUS USB2.0, default mapping: (null)
II 09-01-21 14:44:50.150 - [wayfire-0.6.0/src/core/seat/input-manager.cpp:43] handle new input: Logitech Wireless Mouse, default mapping: (null)
II 09-01-21 14:44:50.150 - [wayfire-0.6.0/src/core/seat/input-manager.cpp:43] handle new input: Asus WMI hotkeys, default mapping: (null)
II 09-01-21 14:44:50.154 - [wayfire-0.6.0/src/core/seat/input-manager.cpp:43] handle new input: AT Translated Set 2 keyboard, default mapping: (null)
II 09-01-21 14:44:50.158 - [wayfire-0.6.0/src/core/seat/input-manager.cpp:43] handle new input: SynPS/2 Synaptics TouchPad, default mapping: (null)
II 09-01-21 14:44:50.158 - [backend/drm/drm.c:1256] Scanning DRM connectors
II 09-01-21 14:44:50.161 - [backend/drm/drm.c:1315] Found connector 'LVDS-1'
II 09-01-21 14:44:50.161 - [backend/drm/drm.c:1351] 'LVDS-1' connected
II 09-01-21 14:44:50.161 - [backend/drm/drm.c:1357] Physical size: 0x0
II 09-01-21 14:44:50.161 - [backend/drm/drm.c:1375] Detected modes:
II 09-01-21 14:44:50.161 - [backend/drm/drm.c:1397]   1366x768@59996 (preferred)
II 09-01-21 14:44:50.161 - [backend/drm/drm.c:1397]   1280x720@59974 
II 09-01-21 14:44:50.161 - [backend/drm/drm.c:1397]   1152x768@59950 
II 09-01-21 14:44:50.161 - [backend/drm/drm.c:1397]   1024x768@59949 
II 09-01-21 14:44:50.161 - [backend/drm/drm.c:1397]   800x600@59960 
II 09-01-21 14:44:50.161 - [backend/drm/drm.c:1397]   848x480@59939 
II 09-01-21 14:44:50.161 - [backend/drm/drm.c:1397]   720x480@59938 
II 09-01-21 14:44:50.161 - [backend/drm/drm.c:1397]   640x480@59938 
II 09-01-21 14:44:50.162 - [backend/drm/drm.c:1315] Found connector 'HDMI-A-1'
II 09-01-21 14:44:50.174 - [backend/drm/drm.c:1315] Found connector 'VGA-1'
II 09-01-21 14:44:50.174 - [backend/drm/drm.c:1463] Requesting modeset for 'LVDS-1'
II 09-01-21 14:44:50.174 - [wayfire-0.6.0/src/core/output-layout.cpp:1101] new output: LVDS-1
II 09-01-21 14:44:50.174 - [backend/drm/drm.c:811] Modesetting 'LVDS-1' with '1366x768@59996mHz'
EE 09-01-21 14:44:50.813 - [wayfire-0.6.0/src/view/xwayland.cpp:497] new xwayland surface (null) class: (null) instance: (null)
II 09-01-21 14:44:50.871 - [wayfire-0.6.0/src/core/seat/input-manager.cpp:43] handle new input: virtual keyboard, default mapping: (null)
EE 09-01-21 14:44:51.302 - [wayfire-0.6.0/src/view/xwayland.cpp:497] new xwayland surface (null) class: (null) instance: (null)
EE 09-01-21 14:44:51.909 - [wayfire-0.6.0/src/view/xwayland.cpp:768] new unmanaged xwayland surface (null) class: (null) instance: (null)
II 09-01-21 14:44:53.584 - [wayfire-0.6.0/src/view/xdg-shell.cpp:199] new xdg_shell_stable surface: (null) app-id: (null)
EE 09-01-21 14:44:56.699 - [wayfire-0.6.0/src/main.cpp:233] Fatal error: Segmentation fault
EE 09-01-21 14:44:56.727 - #1  operator*(glm::mat<4, 4, float, (glm::qualifier)0> const&, wf::point_t const&) ??:?
EE 09-01-21 14:44:56.749 - #2  __restore_rt sigaction.c:?
EE 09-01-21 14:44:56.771 - #3  get_surface_relative_coords(wf::surface_interface_t*, wf::pointf_t const&) ??:?
EE 09-01-21 14:44:56.792 - #4  wf::pointer_t::update_cursor_position(unsigned int, bool) ??:?
EE 09-01-21 14:44:56.813 - #5  wf::pointer_t::handle_pointer_motion(wlr_event_pointer_motion*) ??:?
EE 09-01-21 14:44:56.840 - #6  wf::_safe_list_detail::idle_cleanup_func(void*) ??:?
EE 09-01-21 14:44:56.862 - #7  wlr_region_confine ??:?
EE 09-01-21 14:44:56.888 - #8  wlr_region_confine ??:?
EE 09-01-21 14:44:56.910 - #9  wlr_session_find_gpus ??:?
EE 09-01-21 14:44:56.932 - #10 wl_event_loop_dispatch ??:?
EE 09-01-21 14:44:56.953 - #11 wl_display_run ??:?
EE 09-01-21 14:44:56.980 - #12 main ??:?
EE 09-01-21 14:44:57.006 - #13 __libc_start_main ??:?
EE 09-01-21 14:44:57.034 - #14 _start ??:?
```

**Wayfire version**
0.6.0 on Arch Linux. 
```sh
$ uname -a
Linux zbox 5.10.5-arch1-1 #1 SMP PREEMPT Thu, 07 Jan 2021 09:50:43 +0000 x86_64 GNU/Linux
$ wayfire -v
0.6.0-177fb87 (Jan  8 2021, branch 'master')
```

**My configuration**
https://github.com/zhangkaizhao/configurations/tree/master/wayfire

**Comment by ammen99** [comment]
Hi, I just tried copying your config file but I do not get a crash, even with the animation plugin disabled. Can you elaborate a bit which applications are crashing? Can you try compiling Wayfire with Address Sanitizer? There must have been a memory corruption somewhere, and the backtrace you have does not point to the real problem ...

**Comment by zhangkaizhao** [comment]
Hi @ammen99 ! Thank you for looking into this issue. All applications I tried are crashing, e.g. Alacritty, QTerminal, etc. I am using the packages from AUR https://aur.archlinux.org/packages/wayfire/ (and others). Sorry to hear that the backtrace is irrelevant. I will test it again for the next release.

**Comment by ammen99** [comment]
> Hi @ammen99 ! Thank you for looking into this issue. All applications I tried are crashing, e.g. Alacritty, QTerminal, etc. I am using the packages from AUR https://aur.archlinux.org/packages/wayfire/ (and others). Sorry to hear that the backtrace is irrelevant. I will test it again for the next release.

I actually managed to reproduce the bug, it happens only if the app itself has a close button. Up to now I had only tried closing apps with a shortcut, which doesn't crash. So I think that #1028 has fixed this issue, 0.7.0 will soon be out so I hope it works for you then :) (master should also work)

**Comment by zhangkaizhao** [comment]
Great! <3

**Comment by zhangkaizhao** [comment]
Confirmed: this issue has been fixed after I upgraded Wayfire to 0.7.0. :100:
